torch/ao/quantization/observer.py

```diff
@@ -925,7 +925,7 @@ def __init__(
         quant_min=None,
         quant_max=None,
         factory_kwargs=None,
-        eps=torch.finfo(torch.float32).eps,
+        eps=torch.finfo(torch.float32).eps,  # noqa: B008
```

Comment:
A lot of these B008 are false positives and this rule should definitely remain disabled.